lua/nvim-goc.lua

```diff
@@ -170,14 +170,6 @@ CoverageFunc = function(p, html, customArgs)
   return Coverage(string.gmatch(ts_utils.get_node_text(p)[1], 'Test[^%s%(]+')(), html, customArgs)
```

Comment:
Thanks for the PR, could you also change these two lines (168, 170) to use the new reference?
```suggestion
    return M.CoverageFunc(p, html, customArgs)
  end
  return M.Coverage(string.gmatch(ts_utils.get_node_text(p)[1], 'Test[^%s%(]+')(), html, customArgs)
```

Reply:
Done.